src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1491,6 +1491,13 @@ internal static object GetConverter(Type type, TypeTable backupTypeTable)
                 return typesXmlConverter;
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-linux`
Language Primitive Tests.Test convertion with .Net Core intrinsic type convertor
```powershell
Exception calling "ConvertTo" with "2" argument(s): "Cannot convert the "2,3" value of type "System.String" to type "System.Drawing.Point"."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Api/LanguagePrimitive.Tests.ps1: line 32
32:         $result = [System.Management.Automation.LanguagePrimitives]::ConvertTo('2,3', [System.Drawing.Point])

```
DeserializedMethods.Deserialized objects shouldn't ever have any methods (unless they are primitive known types)
```powershell
Cannot bind parameter 'MemberType'. Cannot convert value "*Method*" to type "System.Management.Automation.PSMemberTypes". Error: "*Method* is not a valid value for PSMemberTypes."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/DeserializedMethods.Tests.ps1: line 19
19:         $d | Get-Member -MemberType *Method* Add | Should -BeNullOrEmpty

```